src/ctx.rs

```diff
@@ -66,6 +66,21 @@ impl Context {
         }
     }
 
+    #[inline]
+    pub fn module_new_from_lookup(&self, alias: &OsStr) -> Result<ModuleIterator> {
+        use std::os::unix::ffi::OsStrExt;
+        let mut list = ptr::null::<kmod_sys::kmod_list>() as *mut kmod_sys::kmod_list;
+        let filename = CString::new(alias.as_bytes())?;
```

Comment:
```suggestion
        let alias = CString::new(alias.as_bytes())?;
```
I think the 2nd argument in `kmod_module_new_from_lookup` is supposed to be an alias instead of a filename

Reply:
yes